common/src/main/java/tachyon/MasterClientBase.java

```diff
@@ -44,6 +48,8 @@
   public MasterClientBase(InetSocketAddress masterAddress, TachyonConf tachyonConf) {
     super(masterAddress, tachyonConf, "master");
     mUseZookeeper = mTachyonConf.getBoolean(Constants.ZOOKEEPER_ENABLED);
+    mCompatibleVersions = new HashSet<Long>();
```

Comment:
How will the `mCompatibleVersions` be maintained? Do you envision it to be specified as constants somewhere, or just hardcoded in each client?